src/components/form-control/form-control.jsx

```diff
@@ -37,7 +37,12 @@ export default function FormControl(props) {
         {props.children}
       </div>
       {isInvalid(props.error, props.readOnly) && (
-        <span id={`${props.id}Hint`} className={styles['error-message']} aria-live="polite" >
+        <span
+          id={`${props.id}Hint`}
+          className={styles['error-message']}
+          aria-live="polite"
+          data-testid="error-message"
```

Comment:
Basically, we will never want to rely on `testid`. We want to our tests to be user-centric: we want to test with queries about _visible_ or _audible_ information (unless I guess it's a ref?)